db/transaction.go

```diff
@@ -158,19 +158,17 @@ func (tx *Transaction) Close() error {
 // Commit tries to commits the changes made to the database in this transaction.
 // If this returns without an error the commit succeeded and the new ref of the
```

Comment:
should prolly update this comment, if no error the ref returned might not be "new"